Intact.BuinessLogic/Mappers/ProtoWarriorMapper.cs

```diff
@@ -19,6 +19,9 @@ public partial class ProtoWarriorMapper
             var model = mapper.Map(d);
             model.Name = localizations.Map(d.TermName, model.Id);
             model.Description = localizations.Map(d.TermDescription, model.Id);
+            model.Abilities = string.IsNullOrWhiteSpace(d.Abilities)
```

Comment:
one more thing , for warriors that don't have abils, would it produce empty collection in json ? I suppose yes, but this would be redundant, I think we could make it nullable and skip deserialization

Reply:
I remade it with the latest change